web_responsive_pwa/controllers/main.py

```diff
@@ -0,0 +1,70 @@
+from odoo.http import request, Controller, route
+
+SERVICE_WORKER_SCRIPT = """
```

Comment:
Instead of harcoding this in python:

1. move this to `service-worker.js` file
2. rename controller to `service-worker-setup.js`
3. make it set only variables (eg: files_to_cache) or render a template like https://github.com/odoo/odoo/blob/13.0/addons/point_of_sale/views/pos_templates.xml#L30
4. use those variables in `service-worker.js`

Reply:
Thanks, done with template:
https://github.com/OCA/web/compare/b16aec8a6424de99027b2f6c2edeb776080d44a7..51bbc19d7008a1cc59c5219d074f5fa293aff646